orte/mca/errmgr/default_hnp/errmgr_default_hnp.c

```diff
@@ -44,6 +49,7 @@
 #include "orte/mca/grpcomm/grpcomm.h"
 #include "orte/mca/ess/ess.h"
 #include "orte/mca/state/state.h"
+#include "orte/orted/pmix/pmix_server.h"
```

Comment:
you have 2 very similar headers pmix_server.h included here. Do you really need both ?

Reply:
i don't need the on in orte/orted, already deleted